jdaviz/configs/mosviz/helper.py

```diff
@@ -659,10 +668,13 @@ def load_images(self, data_obj, data_labels=None, share_image=0):
             different row in the table does not reload the displayed image.
             Currently, if non-zero, the provided number must match the number of
             spectra.
+        add_redshift_column : bool
+            Add redshift column to Mosviz table.
         """
         super().load_data(data_obj, parser_reference="mosviz-image-parser",
                           data_labels=data_labels, share_image=share_image)
-        self._add_redshift_column()
+        if add_redshift_column:
+            self._add_redshift_column()
```

Comment:
Note: Multiple calls to `self._add_redshift_column()` in `load_data` was what causing all the warnings Cami was seeing, so I have to disable this call in sub-calls by default to get rid of the warnings.

Reply:
I confirm I do not see the warnings anymore.